packages/styles/docs/sass.md

```diff
@@ -65,10 +65,20 @@ between version updates.
 
 ## Colors
 
+The colors package will give you access to every color token within the IBM Design Language.
+
+Access these color tokens for your project by using `@use`. For example: 
+
+```scss
+ @use '@carbon/styles/scss/colors';
+```
+
 | Import                               | Filepath            |
 | :----------------------------------- | :------------------ |
 | `@use '@carbon/styles/scss/colors';` | `scss/_colors.scss` |
 
+For more information checkout our [Files](#files) docs.
```

Comment:
To see all the colors available to be imported, checkout our [colors](#todo) docs.